Resources/Prototypes/_EinsteinEngines/NPCs/carp.yml

```
@@ -0,0 +1,82 @@
+    - type: HTN
+      rootTask:
+        task: IdleCompound
+    - type: GhostRole
+      allowMovement: true
+      allowSpeech: true
+      makeSentient: true
+      name: ghost-role-information-space-goldfish-name
+      description: ghost-role-information-space-goldfish-description
+      rules: ghost-role-information-freeagent-rules
+      mindRoles:
+      - MindRoleGhostRoleFreeAgent
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)